packages/backend-common/src/deprecated.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2024 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  resolvePackagePath as _resolvePackagePath,
+  resolveSafeChildPath as _resolveSafeChildPath,
+  isChildPath as _isChildPath,
+} from '@backstage/backend-plugin-api';
+
+import {
+  overridePackagePathResolution as _overridePackagePathResolution,
+  OverridePackagePathResolutionOptions as _OverridePackagePathResolutionOptions,
+  PackagePathResolutionOverride as _PackagePathResolutionOverride,
+} from '@backstage/backend-plugin-api/testUtils';
```

Comment:
Hmm, thinking it might be cleaner to re-export these from `testUtils.ts` instead

Reply:
Done 🙌🏻